nixos/modules/tasks/filesystems/zfs.nix

```diff
@@ -535,5 +546,20 @@ in
         };
       };
     })
+
+    (mkIf cfgTrim.enable {
+      systemd.timers.zpool-trim = {
```

Comment:
Isn't it short to just add `startAt` to the service itself? The timer then gets created automatically.